Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM roland/on-core
```

Comment:
Yes, I think we can create a RackHD organization on docker hub. Then I can change it to be `RackHD/on-core`